sign-in.txt

```diff
@@ -10,4 +10,5 @@
 - Alex Sparks : AlexKSparks
 - Willie Weber, github:weberwilliamb
 - Kevin Durgan, github: ksdurg
+- Sim Ruble : sruble
```

Comment:
You got it Sim - congrats.
Not required, but if you want to practice the process again you could update your entry so it matches the format,
_yourname_, github: _yourgithubname_